dictionary-business/build.gradle.kts

```
@@ -11,9 +11,13 @@ dependencies {
 
     implementation(project(mapOf("path" to ":dictionary-lib-cor")))
     implementation(project(mapOf("path" to ":dictionary-common")))
+    implementation(project(mapOf("path" to ":dictionary-log-common")))
     implementation(project(mapOf("path" to ":dictionary-stubs")))
 
     testImplementation(kotlin("test-junit5"))
     testImplementation("io.kotest:kotest-runner-junit5:$kotestVersion")
     testImplementation("io.kotest:kotest-assertions-core:$kotestVersion")
+    testImplementation(project(mapOf("path" to ":dictionary-repo-in-memory")))
```

Зачем вам здесь мапы?
```suggestion
    testImplementation(project(":dictionary-repo-in-memory"))
```

---

Эту конструкцию прописывает идея самостоятельно по команде добавления зависимости на модуль